packages/ai/cypress/specs/AINoticeIndicator.cy.tsx

```diff
@@ -1,109 +1,162 @@
+		it("Should display correct elements for Shortened mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Shortened');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.should('have.attr', 'icon', 'ai')
-				.should("exist");
+				.should('not.have.attr', 'icon');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-label]")
-				.contains(verificationText)
 				.should("exist");
 		});
 
-		it("Should display correct properties for Shortened mode", () => {
-			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Shortened');
+		it('Should display only the AI icon in IconOnly mode', () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'IconOnly');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
-				.find("[ui5-link]")
-				.should('have.attr', 'icon', '')
+				.find("[ui5-button]")
+				.should('have.attr', 'icon', 'sap-icon://ai')
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
-				.should("exist");
+				.should("not.exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
+				.should("be.empty");
+		});
+	});
+
+	describe("Values of the texts", () => {
+		it("Should display correct values of the properties for Default mode", () => {
+			cy.mount(<AINoticeIndicator></AINoticeIndicator>);
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(verificationText)
-				.should("not.exist");
+				.contains(AI_NOTICE_INDICATOR_ATTRIBUTIONTEXT.defaultText);
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-text]")
+				.contains(AI_NOTICE_INDICATOR_VERIFICATIONTEXT.defaultText);
 		});
 
-		it('Should display only the AI icon in IconOnly mode', () => {
-			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'IconOnly');
+		it("Should display correct values of the properties for Emphasized mode", () => {
+			cy.mount(<AINoticeIndicator mode="Emphasized" attributionText="Made with ai" verificationText="Check the results"></AINoticeIndicator>);
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
-				.find("[ui5-button]")
-				.should('have.attr', 'icon', 'sap-icon://ai')
-				.should("exist");
+				.find("[ui5-link]")
+				.contains("Made with ai");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
+				.contains("Check the results");
+		});
+
+		it("Should display correct values of the properties for Shortened mode", () => {
+			cy.mount(<AINoticeIndicator mode="Shortened" attributionText="Made with ai"></AINoticeIndicator>);
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.should("not.exist");
+				.contains("Made with ai");
+		});
```

Comment:
No need for this test. You already test the verification and attribution texts set by the user in the test above, so this is essentially repeating the checks for attribution text, you can delete it.